assets/src/styles/patterns.scss

```diff
@@ -0,0 +1,2 @@
+// P4 Patterns
+@import "patterns/SideImageWithTextAndCta";
```

Comment:
Patterns are nothing more than copying pasting a collection of blocks inside a post. So it doesn't really make sense to attach custom styles to them.

If the pattern needs to make use of styles, we have the following options:
* Set style options on the blocks before exporting them
* Register a block style and add it to the pattern before exporting to code.